include/net/if.h

```diff
@@ -46,32 +46,48 @@
 
 /* Interface flag bits */
 
-#define IFF_DOWN           (1 << 0) /* Interface is down */
-#define IFF_UP             (1 << 1) /* Interface is up */
-#define IFF_RUNNING        (1 << 2) /* Carrier is available */
-#define IFF_IPv6           (1 << 3) /* Configured for IPv6 packet (vs ARP or IPv4) */
-#define IFF_BOUND          (1 << 4) /* Bound to a socket */
-#define IFF_NOARP          (1 << 7) /* ARP is not required for this packet */
+#define IFF_DOWN           (1 << 0)  /* Interface is down */
+#define IFF_UP             (1 << 1)  /* Interface is up */
+#define IFF_RUNNING        (1 << 2)  /* Carrier is available */
+#define IFF_IPv6           (1 << 3)  /* Configured for IPv6 packet (vs ARP or IPv4) */
+#define IFF_BOUND          (1 << 4)  /* Bound to a socket */
+#define IFF_LOOPBACK       (1 << 5)  /* Is a loopback net */
+#define IFF_POINTOPOINT    (1 << 6)  /* Is point-to-point link */
+#define IFF_NOARP          (1 << 7)  /* ARP is not required for this packet */
+#define IFF_MULTICAST      (1 << 12) /* Supports multicast. */
+#define IFF_BROADCAST      (1 << 13) /* Broadcast address valid. */
```

Comment:
Just a question: why 12 and 13 bits are used and not 8 and 9?

Reply:
This is still not answered.

Reply:
> Just a question: why 12 and 13 bits are used and not 8 and 9?

I was just reusing the Linux definition, do I need to change it to 8 and 9?

Reply:
> This is still not answered.

Sorry for my reply are not timely.

Reply:
Hello @zhhyu7

Sorry for my delay. I was away for a couple of days. The PR looks good. I will merge it now. If there is ever a need to change the bit values, they can always be changed.